GiniCapture/Classes/Core/Screens/Camera/CameraViewController.swift

```diff
@@ -236,6 +236,27 @@ extension CameraViewController {
         self.toolTipView?.alpha = 0
     }
     
+    /**
```

Comment:
This sneaked in from https://github.com/gini/gini-vision-lib-ios/tree/PIA-711 and causes a build error.

Reply:
If I comment out this unused code, then it builds and works fine.

Reply:
I need to practice more patches )